pkgs/tools/misc/lbdb/default.nix

```diff
@@ -7,15 +7,17 @@
 }:
 
 let
-  version = "0.48.1";
+  version = "0.48.2";
 in
 with stdenv.lib;
 with perlPackages;
 stdenv.mkDerivation {
   name = "lbdb-${version}";
-  src = fetchurl {
-    url = "http://www.spinnaker.de/lbdb/download/lbdb_${version}.tar.gz";
-    sha256 = "1gr5l2fr9qbdccga8bhsrpvz6jxigvfkdxrln9wyf2xpps5cdjxh";
+  src = fetchFromGitHub {
+		owner = "bfortz";
+		repo = "lbdb";
+		rev = "${version}";
+    sha256 = "1ldh82pzm4677wyxxqi18l8m1cm3n5sbllscx4pjnjwza8hg9g4m";
```

Comment:
please fetch only the corresponding patch:
```diff
diff --git c/pkgs/tools/misc/lbdb/default.nix i/pkgs/tools/misc/lbdb/default.nix
index a00da33d8c9..c915f56904a 100644
--- c/pkgs/tools/misc/lbdb/default.nix
+++ i/pkgs/tools/misc/lbdb/default.nix
@@ -32,7 +32,14 @@ stdenv.mkDerivation {
     ++ optional   (khard != null) "--with-khard"
     ++ optional      (mu != null) "--with-mu";
 
-  patches = [ ./add-methods-to-rc.patch ];
+  patches = [ ./add-methods-to-rc.patch
+    # fix undefined exec_prefix. Remove with the next release
+    (fetchpatch {
+      url = "https://github.com/RolandRosenfeld/lbdb/commit/60b7bae255011f59212d96adfbded459d6a27129.patch";
+      sha256 = "129zg086glmlalrg395jq8ljcp787dl3rxjf9v7apsd8mqfdkl2v";
+      excludes = [ "debian/changelog" ];
+    })
+  ];
   postFixup = "wrapProgram $out/lib/mutt_ldap_query --prefix PERL5LIB : "
     + "${AuthenSASL}/${perl.libPrefix}"
     + ":${ConvertASN1}/${perl.libPrefix}"
```
and the commit message should be `lbdb: fix build`. Besides that, LGTM

Reply:
Thanks for this, clean solution! (I'm learning every day...)